withings_api/common.py

```diff
@@ -213,10 +214,11 @@ class UserGetDeviceResponse(NamedTuple):
     devices: Tuple[UserGetDeviceDevice, ...]
 
 
-class SleepGetTimestamp(NamedTuple):
-    """SleepGetTimestamp."""
+class SleepGetTimestampValue(NamedTuple):
```

Comment:
Let's keep the class name `SleepGetTimestamp`. Doing so will avoid a breaking change to the API.

Reply:
I am always for not breaking the API. In this case it is broken either way, isn't it? 

- if SleepGetTimestamp is not used, the change should not matter
- it is only used for hr/rr in Sleep Get, which afaict never worked (actually requesting this data caused an exception, see #38, so I would expect that nobody is using it?)
- if we keep the name but add a second field, use of the class is broken anyway (you would get `TypeError: __new__() missing 1 required positional argument: 'value')

Only example where I would expect that existing working code is affected: somebody uses `from withings_api import SleepGetTimestamp` but doesn't actually use it. Still, this would stop their program from running but is easy to fix on their side. We could also prevent it by keeping the original SleepGetTimestamp?

What do you think?

Reply:
Interesting that `$timestamp` is no longer present, it was working a few months ago. This wouldn't be the first time Withings changed the API.
Thanks for reminding me the NamedTuples are intended for read only use. Hence the existence of the helper methods.